### PR TITLE
Added icons specified in 603

### LINKED ISSUE #603: Add two new icons
Add the attached icons into the Lucid icon library.

### PR DESCRIPTION
**Pull request by SirKitboard**
## PR Checklist

- Manually tested across supported browsers
  - [X] Chrome
  - [X] Firefox
  - [x] Safari
  - [x] IE11 (Win 7)
  - [x] Edge (Win 10)
- [x] Unit tests written (`common` at minimum)
- [x] PR has one of the `semver-` labels
- [x] Two core team engineer approvals
- [x] One core team UX approval

Fixes #603 

Docs uploaded here: http://docspot.devnxs.net/projects/lucid/603-add-new-icons/

### CODE REVIEW

**Comment by jondlm** [comment]
@SirKitboard Thanks for your contributions! We're working today on the issue of travis builds failing for personal forks.

File: src/components/Icon/AddURLIcon/AddURLIcon.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+			...passThroughs,
+		} = this.props;
+
+		return (
+			<Icon
+				{...passThroughs}
+				{..._.pick(passThroughs, _.keys(Icon.propTypes))}
+				className={cx('&', className)}
+			>
+				<path d="M15.893,11.447c0.102,0.109,0.133,0.271,0.087,0.418c-0.06,0.141-0.189,0.252-0.347,0.268l-2.08,0.224l1.018,2.524c0.04,0.1,0.042,0.217,0,0.314s-0.127,0.179-0.227,0.219l-1.383,0.559c-0.199,0.081-0.452-0.027-0.533-0.227l-1.02-2.524l-1.736,1.354c-0.124,0.096-0.28,0.113-0.42,0.054c0,0-0.016-0.007-0.023-0.026C9.088,14.542,8.998,14.404,9,14.24l0.005-8.994C9.01,5.082,9.108,4.925,9.268,4.861c0.16-0.064,0.342-0.021,0.458,0.094L15.893,11.447z M1.678,1C0.751,1,0,1.751,0,2.679v8.644C0,12.249,0.751,13,1.678,13H8v-1.983H2.319c-0.185,0-0.335-0.15-0.335-0.335V3.336C1.983,3.15,2.134,3,2.319,3H16V1H1.678z"/>
```
Comment (ogupte) [inline src/components/Icon/AddURLIcon/AddURLIcon.jsx]: these need to be single quotes

File: src/components/Icon/UploadIcon/UploadIcon.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+			...passThroughs,
+		} = this.props;
+
+		return (
+			<Icon
+				{...passThroughs}
+				{..._.pick(passThroughs, _.keys(Icon.propTypes))}
+				className={cx('&', className)}
+			>
+				<path d="M4.288,6L3,5c0,0,3.751-4.379,3.754-4.377C7.283,0,8,0,8,0s0.717,0,1.246,0.623C9.249,0.621,13,5,13,5l-1.288,1L9,2.729V10c0,0.553-0.447,1-1,1s-1-0.447-1-1V2.729L4.288,6z M15,12v1.75c0,0.138-0.112,0.25-0.25,0.25H1.25C1.112,14,1,13.888,1,13.75V12H0v2.75C0,15.44,0.56,16,1.25,16h13.5c0.69,0,1.25-0.56,1.25-1.25V12H15z"/>
```
Comment (ogupte) [inline src/components/Icon/UploadIcon/UploadIcon.jsx]: single quotes here too

**Comment by mute** [comment]
Fixes #603

**Pull request merged**